README.md

```diff
@@ -102,48 +101,8 @@ Beyond the community, you can also read [customer stories][stories] for commerci
 
 [stories]: https://authzed.com/customers
 
-## Joining the Community
```

Comment:
I do think we should have a Discord reference at the top

Reply:
There is - it's one of the icons